content/2015/07/manual-review-best-practices-whats-the-value-of-assessing-ip-address-risk.md

```diff
@@ -42,7 +42,7 @@ as well as to employees who work remotely or are traveling.
 
 To provide a deeper understanding of IP address risk,
 [minFraud](https://www.maxmind.com/en/minfraud-services) generates a
-_proxyScore\*_ for each transaction. The proxyScore indicates how likely it is
+\*proxyScore\*\* for each transaction. The proxyScore indicates how likely it is
```

Comment:
Just out of curiosity because it caught my eye. Were these extra asterisks on purpose ? Because this will show up as `*proxyScore**`, which won't match the asterisk in the footer. 
<img width="209" alt="image" src="https://user-images.githubusercontent.com/104642023/207735139-680c37e3-42f3-449b-9e3b-c7cf08b27644.png">

Reply:
I'm talking about this one 
<img width="593" alt="image" src="https://user-images.githubusercontent.com/104642023/207735397-fb112436-0ab6-4ff0-88e0-dad8edabec46.png">

Reply:
`prettier` did this. I am not sure why. I agree that it looks wrong.